packages/app/src/components/SettingsModal.tsx

```diff
@@ -135,6 +137,22 @@ export const GeneralSettingsPage: FC = () => {
           />
         )}
       </Field>
+      <Field name="preserve-port-text-case" label="Preserve text case for node ports">
+        {() => (
+          <>
+            <Toggle
+            id="check-for-updates"
+            isChecked={preservePortTextCase}
+            onChange={(e) => {
+              setPreservePortTextCase(e.target.checked);
+            }}
+            />  
+            <HelperMessage>
+              This WILL preserve the text fromat of the node port names.  e.g. `newInputPort` will be shown instead of `NEWINPUTPORT` when enabled
```

Comment:
```suggestion
              This WILL preserve the text format of the node port names.  e.g. `newInputPort` will be shown instead of `NEWINPUTPORT` when enabled
```